proxy/cache.go

```diff
@@ -20,7 +20,7 @@ type cacheValue struct {
 }
 
 func newCache(size int) (*cache, error) {
-	c, err := specialized.NewCache(size)
+	c, err := specialized.NewCache(size, true)
```

Comment:
Can we make this a flag with a default value of false?

Reply:
Done.